src/main/java/frc/robot/RobotContainer.java

```diff
@@ -128,6 +138,14 @@ public RobotContainer() {
       }
     }
 
+    arm =
```

Comment:
This should be moved into the cases, in order to ensure that sim ignores it (will fix this in a commit soon)